### ISSUE TITLE
A question about pre-trained weights

### ISSUE THREAD

**Issue by indersingh17188** [SlongLiu/ASL_reproduce#1]
Hello,

Nice re-implementation with proper log information tracking and a lot of tricks. Appreciate your additional efforts in addition to the original authors's work. However, I have a question regarding the default pre-trained weights which you have used to initialize the training. For example: the one which you have used locally were I guess: '/data/shilong/data/pretrained/tresnet_l_448.pth'. Are they the one which are shared in the repo at the link https://github.com/Alibaba-MIIL/ASL/blob/main/MODEL_ZOO.md

When I am starting the training from scratch without any pre-trained weights, my loss is extremely high and the val mAP is never crossing 50 or so even after 80 epochs. 

Thanks in advance for your time. 

Best,
Inder
